lib/util/http/azure.ts

```diff
@@ -0,0 +1,79 @@
+import is from '@sindresorhus/is';
+import type { PagedResult } from '../../modules/platform/azure/types';
+import type { InternalJsonUnsafeOptions } from './http';
+import { HttpBase } from './http';
+import type { HttpMethod, HttpOptions, HttpResponse } from './types';
+
+const MAX_LIMIT = 100;
+
+let baseUrl: string;
+export const setBaseUrl = (url: string): void => {
```

Comment:
use normal function declaration instead of lambda

Reply:
done in the other PR